packages/@aws-cdk/aws-amplify/lib/index.ts

```diff
@@ -1,2 +1,7 @@
 // AWS::Amplify CloudFormation Resources:
 export * from './amplify.generated';
+export * from './app';
+export * from './branch';
+export * from './buildspec';
+export * from './domain';
+export * from './shared';
```

Comment:
Do not export "shared", seems like all of them should be internal. We are also standardizing around naming unexported files with an underscore prefix (`_shared.ts`).